projects/cdk/directives/for-async/for-async.directive.ts

```diff
@@ -15,7 +15,7 @@ export class TuiForAsyncDirective<T> implements OnChanges, OnDestroy {
     private readonly destroy$ = new Subject<void>();
 
     @Input()
-    tuiForAsyncOf: readonly T[] | null | undefined;
+    tuiForAsyncOf: T[] | readonly T[] | null | undefined;
```

Comment:
It is strange behaviour 🤔 
`readonly T[]` should accepts `T[]`.

This example don't throws any error. Which typescript version you use ?
```ts
let a: ReadonlyArray<string> = [];
const b: string[] = ['1', '2'];

a = b;
```

Reply:
Yes, but I don't understand why IDEA doesn't resolve it

Reply:
Maybe instead write `<T extends readonly any[]>` or something like that?

Reply:
fixed